peter-piper.js

```diff
@@ -22,4 +22,11 @@ var counter = 0;
 // TODO: Count the number of times an uppercase P or lowercase p character
```

Comment:
This looks good, Christian! Could you remove these two comments, though, since they are no longer correct?